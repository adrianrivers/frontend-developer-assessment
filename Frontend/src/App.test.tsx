// import React from 'react'
// import { render } from '@testing-library/react'
// import App from './Template'

// test('renders the footer text', () => {
//   const { container } = render(<App />)

//   // const footerElement = screen.getByText(/clearpoint.digital/i)
//   // expect(footerElement).toBeInTheDocument()

//   expect(container).toMatchSnapshot()
// })
