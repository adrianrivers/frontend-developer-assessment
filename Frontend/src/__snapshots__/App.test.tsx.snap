// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the footer text 1`] = `
<div>
  <div
    class="flex flex-col h-screen justify-between"
  >
    <main
      class="h-100 w-full flex items-center justify-center bg-teal-lightest"
    >
      <div
        class="bg-white p-6 m-4 w-full"
      >
        <div
          class="mb-4"
        >
          <form>
            <div
              class="mb-4"
            >
              <div>
                <label
                  for="Add a todo item-input"
                >
                  Add a todo item
                  :
                </label>
                <input
                  class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
                  id="Add a todo item-input"
                  name="Add a todo item"
                  pattern="[a-z]{3,10}"
                  placeholder="Description..."
                  required=""
                  type="text"
                />
              </div>
            </div>
            <button
              class="border rounded mt-4 px-6 h-10"
            >
              Add
            </button>
          </form>
        </div>
      </div>
    </main>
    <footer
      class="p-4 bg-white md:px-6 md:py-8 dark:bg-gray-800"
    >
      <hr
        class="my-6 sm:mx-auto dark:border-gray-700 lg:my-8"
      />
      <span
        class="block text-sm text-gray-500 sm:text-center dark:text-gray-400"
      >
        Â© 2022 Copyright:
         
        <a
          class="hover:underline"
          href="https://flowbite.com"
        >
          clearpoint.digital
        </a>
      </span>
    </footer>
  </div>
</div>
`;
